compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -1033,37 +1039,30 @@ execMigrate projectOpts inFile1_ inFile2_ mbDir =
                   let pkgName = takeBaseName $ mainDalfPath dalfManifest
                   mainDalfEntry <- getEntry (mainDalfPath dalfManifest) dar
                   (mainPkgId, mainLfPkg) <- decode $ BSL.toStrict $ ZipArchive.fromEntry mainDalfEntry
-                  pure (pkgName, mainPkgId, mainLfPkg, sdkVersion dalfManifest)
+                  pure (pkgName, mainPkgId, mainLfPkg)
           -- generate upgrade modules and instances modules
           let eqModNames =
                   (NM.names $ LF.packageModules lfPkg1) `intersect`
                   (NM.names $ LF.packageModules lfPkg2)
-          thisSdkVer <- getSdkVersion
-          let differentSdks = (length $ nubSort [sdkVer1, sdkVer2, thisSdkVer]) /= 1
           let eqModNamesStr = map (T.unpack . LF.moduleNameString) eqModNames
-          let buildOptions0 =
-                  [ "--allow-different-sdks"
-                  , "'--package=" <> show ("instances-" <> pkgName1
-                                          , [(m', m' ++ "A")
-                                            | m <- eqModNamesStr, let m' = m <> "Instances"
+          let buildOptions =
+                  ["'--package=" <> show ("instances-" <> pkgName1
+                                          , [(m', m'')
+                                            | m <- eqModNamesStr
+                                            , let m' = m <> "Instances"
```

Comment:
I feel like these arguments would be easier if you inline it into the tuple, i..e,
```
[(m <> "Instances", m <> "AInstances") | m <- eqModNamesStr]
```

Reply:
yes, why not!